tests/integration/test_fim/test_registry/test_checks/test_check_checksum.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Comment:
Same as with `test_check_all.py`, tests are pretty similar between them, could they be unified? Going a step further, could `test_check_all` and `test_check_checksum` be unified between them?